sdk/core/azure-core-tracing-opentelemetry/src/samples/CreateConfigurationSettingTracingSample.md

```diff
@@ -53,11 +54,16 @@ public class Sample {
 
     private static Tracer configureOpenTelemetryAndLoggingExporter() {
         LoggingSpanExporter exporter = new LoggingSpanExporter();
-        OpenTelemetrySdk openTelemetry = OpenTelemetrySdk.builder().build();
-        openTelemetry
-            .getTracerManagement()
-            .addSpanProcessor(SimpleSpanProcessor.builder(exporter).build());
-        return openTelemetry.getTracer("Sample");
+        // Tracer provider configured to export spans with SimpleSpanProcessor using
```

Comment:
Can these samples use `embedme` to ensure these are compilable?

Reply:
Sure, will cover in this issue https://github.com/Azure/azure-sdk-for-java/issues/13423. The only holding issue with this being compileable, is the dependencies that we will need to add to this package (even if test).

Should that be fine?

Reply:
Sure, that works.